pkg/daemonset/daemonset.go

```diff
@@ -138,6 +138,71 @@ func (builder *Builder) WithNodeSelector(selector map[string]string) *Builder {
 	return builder
 }
 
+// WithHostNetwork applies HostNetwork to pod's definition.
+func (builder *Builder) WithHostNetwork() *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Enabling hostnetwork to daemonset %s in namespace %s",
```

Comment:
Please change log message to: "Applying HostNetwork flagdaemonset %s in namespace %s"

Reply:
updated as suggested